mint.json

```diff
@@ -40,6 +40,14 @@
       "name": "Components",
       "url": "content"
     },
+    {
+      "name": "Advanced",
+      "url": "advanced"
+    },
+    {
+      "name": "Mintlify API",
+      "url": "api-reference"
+    },
```

Comment:
A bit weird imo to bundle chat into api reference and also with the API reference not being super robust. Can we move these into advanced?

Reply:
Chatted with @hahnbeelee about this - the widget is technically an "API". I initially gave widget its own tab but that looked weird with only one page. Happy to just place this in advanced for now until we have more widget/api features, but then I do think we should separate these two out and give them a dedicated tab